src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,26 @@
+package lotto.controller;
+
+import lotto.domain.Budget;
+import lotto.domain.Lotto;
+import lotto.domain.LottoGenerator;
+import lotto.domain.LottoStatisticComputer;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.List;
+
+public class LottoController {
+
+    private LottoGenerator lottoGenerator = new LottoGenerator();
+    private LottoStatisticComputer lottoStatisticComputer;
+
+
+    public void buyLotto() {
+        Budget budget = InputView.inputBuyBudget();
+        List<Lotto> lottos = lottoGenerator.generateLottos(budget);
```

Comment:
```suggestion
        Lotto lottos = lottoGenerator.generateLottos(budget);
```

`Lotto` 객체를 바로 만드는건 어떤가요?

Reply:
`Lotto::생성해줘(...)` 
객체스스로 생성할수 있을것 같은데 `generator`가 있어야할까요? 🤔

Reply:
작성하면서도 뭔가 이상하단 생각이 들었는데 이게 큰 원인이었던거같네요.😱